src/Sql/Sql/help/New-AzSqlInstanceLink.md

```diff
@@ -15,97 +15,101 @@ Creates a new instance link.
 ```
 
-```output		
-ResourceGroupName              : ResourceGroup01
-InstanceName                   : ManagedInstance01
-Type                           : Microsoft.Sql/managedInstances/distributedAvailabilityGroups
-Id                             : /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/ResourceGroup01/providers/Microsoft.Sql/managedInstances/ManagedInstance01/distributedAvailabilityGroups/Link01
-Name                           : Link01
-TargetDatabase                 : Database01
-SourceEndpoint                 : TCP://SERVER01:5022
-PrimaryAvailabilityGroupName   : Link01PrimaryAG
-SecondaryAvailabilityGroupName : Link01SecondaryAG
-ReplicationMode                : Async
-DistributedAvailabilityGroupId : xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
-SourceReplicaId                : xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
-TargetReplicaId                : xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
-LinkState                      : Copying
-LastHardenedLsn                :
+```output
+ResourceGroupName                : ResourceGroup01
+InstanceName                     : ManagedInstance01
+Type                             : Microsoft.Sql/managedInstances/distributedAvailabilityGroups
+Id                               : /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/ResourceGroup01/providers/Microsoft.Sql/managedInstances/ManagedInstance01/distributedAvailabilityGroups/Link01
+Name                             : Link01
+DistributedAvailabilityGroupName : Link01
+DistributedAvailabilityGroupId   : xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+Databases                        : {Database01}
+InstanceAvailabilityGroupName    : AG_Database01_MI
+PartnerAvailabilityGroupName     : AG_Database01
+PartnerEndpoint                  : TCP://SERVER01:5022
+InstanceLinkRole                 : Secondary
+PartnerLinkRole                  : Primary
+ReplicationMode                  : Async
+FailoverMode                     : Manual
+SeedingMode                      : Automatic
 ```
 
 This command creates a new instance link with name "Link01".
 
 ### Example 2: Create a new instance link using an instance object
 ```powershell
 $instance = Get-AzSqlInstance -ResourceGroupName "ResourceGroup01" -Name "ManagedInstance01"
-New-AzSqlInstanceLink -InstanceObject $instance -Name "Link01" -PrimaryAvailabilityGroupName "Link01PrimaryAG" -SecondaryAvailabilityGroupName "Link01SecondaryAG" -TargetDatabase "Database01" -SourceEndpoint "TCP://SERVER01:5022"
+New-AzSqlInstanceLink -InstanceObject $instance -Name "Link01" -Database @("Database01") -InstanceAvailabilityGroupName "AG_Database01_MI" -PartnerAvailabilityGroupName "AG_Database01" -InstanceLinkRole "Secondary" -PartnerEndpoint "TCP://SERVER01:5022" -FailoverMode "Manual" -SeedingMode "Automatic"
```

Comment:
Currently, on the backend we allow only 1 value in the list. But in near future we are planning to change that, so backend will accept 1 or more. So I would leave this as it is right now. And when we enable that in the backend I will change the examples.

Reply:
If you only need to pass in one value, then the example should be `New-AzSqlInstanceLink -InstanceObject $instance -Name Link01 -Database Database01 -InstanceAvailabilityGroupName "AG_Database01_MI" ...` instead of `-Database @("Database01")`